api-docs/api-reference/methods/post-create-private-monitor-zone.rst

```diff
@@ -0,0 +1,73 @@
+
++--------------------------+-------------------------+-------------------------+
+|Response Code             |Name                     |Description              |
++==========================+=========================+=========================+
+|201                       |OK                       |The 'Location' header    |
+|                          |                         |contains a link to the   |
+|                          |                         |newly created entity.    |
++--------------------------+-------------------------+-------------------------+
+|400                       |Bad request              |The system received an   |
+|                          |                         |invalid value in a       |
+|                          |                         |request.                 |
++--------------------------+-------------------------+-------------------------+
+|401                       |Unauthorized             |The system received a    |
+|                          |                         |request from a user that |
+|                          |                         |is not authenticated.    |
++--------------------------+-------------------------+-------------------------+
+|403                       |Forbidden                |The system received a    |
+|                          |                         |request that the user is |
+|                          |                         |not authorized to make.  |
++--------------------------+-------------------------+-------------------------+
+|500                       |Internal Server Error    |An unexpected condition  |
+|                          |                         |was encountered.         |
++--------------------------+-------------------------+-------------------------+
+|503                       |Service Unavailable      |The system is            |
+|                          |                         |experiencing heavy load  |
+|                          |                         |or another system        |
+|                          |                         |failure.                 |
++--------------------------+-------------------------+-------------------------+
+Request
+-------
+
+The following table shows the header parameters for the request:
+
++-----------------+----------------+-----------------------------------------------+
+|Name             |Type            |Description                                    |
++=================+================+===============================================+
+|X-Auth-Token     |String          |A valid authentication token with              |
+|                 |*(Required)*    |administrative access. For details, see        |
+|                 |                |:ref:`Get your credentials <get-credentials>`  |
++-----------------+----------------+-----------------------------------------------+
+
+.. note:: This operation does not accept a request body.
```

Comment:
Don't need this note since the request does have a body.

Reply:
...and the body only requires a `label` field of type string that is the same concept as labels on our other objects.

Reply:
Removed note and added sentence under the Example title about the ```label``` field.